src/models/__tests__/helpers/mockCompanyHouse.ts

```diff
@@ -0,0 +1,64 @@
+import { container } from 'tsyringe'
+import { Dispatcher, MockAgent, getGlobalDispatcher, setGlobalDispatcher } from 'undici'
+import { Env } from '../../../env'
+
+const env = container.resolve(Env)
+
+export const validCompanyNumber = '10592650'
+export const invalidCompanyNumber = '105926502'
+export const noCompanyNumber = ''
+
+export const successResponse = {
+  registered_office_address: {
+    address_line_1: '51 Mornington Crescent',
+    locality: 'Hounslow',
+    postal_code: 'TW5 9ST',
+    country: 'United Kingdom',
+  },
+  company_status: 'dissolved',
+  registered_office_is_in_dispute: false,
+  company_name: 'SMH IOT SOLUTIONS LTD',
+  company_number: '10592650',
+}
```

Comment:
maybe create something like `fixtures/` folder for this kinda stuff but again, is just 10 lines so I would not worry too much

Reply:
Noted, I will find an example and follow, I'm sure as we go along there will be many more fixtures.